match2/commons/match_group_legacy.lua

```diff
@@ -172,7 +172,7 @@ function p._convert(mapping)
 		end
 
 		if not Logic.isEmpty(match) then
-			if index ~= 'RxMBR' then
+			if index ~= "RxMBR" and index ~= "RxMTP" then
```

Comment:
It would be nice to use constants to explain these string literals, they're quite cryptic.